### PR TITLE
Minor improvements and bug fixes in bc-explorer tui

### PR DESCRIPTION
**Pull request by YoshihitoAso**
- Some IDE environments were causing module import errors, so I reviewed the import way.
- Fixed some minor bugs.

### CODE REVIEW

File: cmd/explorer/src/main.py [hunk 1]
```diff
@@ -17,22 +17,18 @@
...
 app = typer.Typer(pretty_exceptions_show_locals=False)
 
 
 @app.command()
 def run(
-    url: str = typer.Argument("http://localhost:5000", help="ibet-Wallet-API server URL to connect"),
-    lot_size: int = typer.Argument(100, help="Lot size to fetch Block Data list"),
+    url: str = typer.Option("http://localhost:5000", help="ibet-Wallet-API server URL to connect"),
+    lot_size: int = typer.Option(100, help="Lot size to fetch Block Data list"),
 ):
```
Comment (YoshihitoAso) [inline cmd/explorer/src/main.py:32]: In this case, I think it should be Option.

File: cmd/explorer/src/gui/widget/query_panel.py [hunk 2]
```diff
@@ -235,7 +235,7 @@ async def on_button_pressed(self, event: Button.Pressed) -> None:
         Occurs when `Button.Pressed` is emitted.
         """
         if event.button.id == ID.QUERY_PANEL_ENTER:
-            await self.action_enter()
+            self.action_enter()
```
Comment (YoshihitoAso) [inline cmd/explorer/src/gui/widget/query_panel.py:238]: There was a problem when clicking the button.

File: cmd/explorer/src/gui/screen/block.py [hunk 3]
```diff
@@ -116,25 +138,65 @@ async def on_button_pressed(self, event: Button.Pressed) -> None:
...
+    def reload_block(self) -> None:
+        if self.tui.state.current_block_number is None or self.tui.state.block_list_query is None:
+            return
+
+        self.query_one(BlockListQueryPanel).block_list_query = self.tui.state.block_list_query
+        asyncio.create_task(self.fetch_block_list())
+
+    ##################################################
+    # Key biding
+    ##################################################
```
Comment (purplesmoke05) [inline cmd/explorer/src/gui/screen/block.py:185]: “binding” typo I think.

**Comment by purplesmoke05** [comment]
Appending path in `connector` can also be removed.

https://github.com/BoostryJP/ibet-Wallet-API/blob/6050dd3bdac1bd4f9bbae78db873377374a61a34/cmd/explorer/src/connector/__init__.py#L19-L28

**Review by purplesmoke05 — APPROVED** [review]
This completely solved IDE import problems🤯
Awesome.

**Comment by purplesmoke05** [comment]
approved including https://github.com/BoostryJP/ibet-Wallet-API/pull/1343/commits/6ab5b53065d6abd42ae5d8ab69048801666b66fe

**Pull request merged**